### PR TITLE
docs(README): changes proposal

### PR DESCRIPTION
**Pull request by GregoryIvo**
This PR aims to make Kura's readme show off the amazing features it offers to commuters. 

> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

Brief description of the PR. [e.g. Added `null` check on `object` to avoid `NullPointerException`]

**Related Issue:** This PR fixes/closes {issue number}

**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped

**Screenshots:** If applicable, add screenshots to help explain your solution

**Manual Tests**: Optional description of the tests performed to check correct functioning of changes, useful for an efficient review

**Any side note on the changes made:** Description of any other change that has been made, which is not directly linked to the issue resolution [e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,24 @@ Eclipse Kura™
...
+## What can Eclipse-Kura do for me?
+* **Kura-Services:** Provision and set up features to run on your gateway, such as an MQTT broker.
+* **Kura-Networking:** Manage Network connectivity, including 
+* **Kura-Wires:** Design data flows and data processing streams effortlessly with a drag-and-drop visual editor.
+* **Kura-Cloud-Connectors:** Extendable cloud connector system. 
+* **Kura-Drivers:** Extendable service that handles reading data off of external devices.
+* **Kura-Snapshots:** Securely store and re-apply gateway settings for convenience.
+* **Kura-Security**: Easily and safely store your secrets.
+* **Kura-Container-Orchestrator**: Manage Docker or Podman containers on your gateway for ultimate flexibility.
+* **Kura-AI-Infrence**: Run Nvidia Triton Models on the edge.
```
Comment (salvatore-coppola) [inline README.md]: typo in inference

File: README.md [hunk 2]
```diff
@@ -15,11 +15,24 @@ Eclipse Kura™
...
+ 
+## I have used Eclipse-Kura to make a small-scale Edge deployment, how do I scale now?
+If you want to scale, and manage many instances of Eclipse-Kura, check out Eclipse-Kapua. Eclipse-Kapua is a Eclipse-Kura compatible cloud command and control service that allows you to aggregate data and configure many Eclipse-Kura devices.
```
Comment (salvatore-coppola) [inline README.md]: i'd add a link to kapua repository

**Comment by salvatore-coppola** [comment]
I would add other items about the rest apis and java apis to emphasise the programmability of the platform

File: README.md [hunk 3]
```diff
@@ -15,11 +15,24 @@ Eclipse Kura™
...
-
-Kura aims at offering a Java/OSGi-based container for M2M applications running in service gateways. Kura provides or, when available, aggregates open source implementations for the most common services needed by M2M applications. Kura components are designed as configurable OSGi Declarative Service exposing service API and raising events. While several Kura components are in pure Java, others are invoked through JNI and have a dependency on the Linux operating system.
-
-For more information, see the [Eclipse project proposal](http://www.eclipse.org/proposals/technology.kura/).
+## What is Eclipse Kura?
+From [the maori word for tank/container](https://maoridictionary.co.nz/search/?keywords=kura), Eclipse Kura is a versatile software framework designed to supercharge your edge devices. With an intuitive web interface, Eclipse Kura streamlines the process of configuring your gateway, connecting sensors, and IoT devices to seamlessly collect, process, and send data to the cloud.
+ 
+Eclipse Kura runs on an edge gateway, which can be anything from a small SBC(single-board computer) like a Raspberry Pi, or a powerful high-performance computer.
+
+## What can Eclipse-Kura do for me?
```
Comment (mattdibi) [inline README.md]: Remove the unnecessary `-` all over the place. Add the trademark.

```suggestion
## What can Eclipse Kura™ do for me?
```

File: README.md (hunk 1 above)
Comment (mattdibi) [inline README.md]: `-` everywhere

```suggestion
* **Kura Services:** Provision and set up features to run on your gateway, such as an MQTT broker.
* **Kura Networking:** Manage Network connectivity, including 
* **Kura Wires:** Design data flows and data processing streams effortlessly with a drag-and-drop visual editor.
* **Kura Cloud Connectors:** Extendable cloud connector system. 
* **Kura Drivers:** Extendable service that handles reading data off of external devices.
* **Kura Snapshots:** Securely store and re-apply gateway settings for convenience.
* **Kura Security**: Easily and safely store your secrets.
* **Kura Container Orchestrator**: Manage Docker or Podman containers on your gateway for ultimate flexibility.
* **Kura AI Inference**: Run Nvidia Triton Models on the edge.
```

Comment (mattdibi) [reply README.md]: Also... link each of these to the relevant documentation section?

**Review by mattdibi — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -15,18 +15,33 @@ Eclipse Kura™
...
-
-For more information, see the [Eclipse project proposal](http://www.eclipse.org/proposals/technology.kura/).
+## What is Eclipse Kura™?
+From [the maori word for tank/container](https://maoridictionary.co.nz/search/?keywords=kura), Eclipse Kura™ is a versatile software framework designed to supercharge your edge devices. With an intuitive web interface, Eclipse Kura™ streamlines the process of configuring your gateway, connecting sensors, and IoT devices to seamlessly collect, process, and send data to the cloud. Eclipse Kura™ provides an extensible Java API for developing custom plugins within the framework. Additionally, it offers a REST API, enabling the use of Eclipse Kura™ as a backend service in your application.
+ 
+Eclipse Kura™ runs on an edge gateway, which can be anything from a small SBC(single-board computer) like a Raspberry Pi, or a powerful high-performance computer.
+
+## What can Eclipse Kura™ do for me?
```
Comment (mattdibi) [inline README.md]: I would set this as an H3 or H4 header just so it's clear that the first section of the readme is dedicated to explaining what is Kura

```suggestion
### What can Eclipse Kura™ do for me?
```

File: README.md [hunk 5]
```diff
@@ -15,18 +15,33 @@ Eclipse Kura™
...
+* **Kura™ Cloud Connectors:** Extendable cloud connector system. 
+* **Kura™ Drivers:** Extendable service that handles reading data off of external devices.
+* **Kura™ Snapshots:** Securely store and re-apply gateway settings for convenience.
+* **Kura™ Security**: Easily and safely store your secrets.
+* **Kura™ Container Orchestrator**: Manage Docker or Podman containers on your gateway for ultimate flexibility.
+* **Kura™ AI Inference**: Run Nvidia Triton Models on the edge.
+* **Kura™ Plugins**: Add and Extend the framework by adding your own Services, and Drivers.
+* **Kura™ REST Service**: Embed the framework as a backend in your own edge applications.
+ 
+## I have used Eclipse Kura™ to make a small-scale Edge deployment, how do I scale now?
```
Comment (mattdibi) [inline README.md]: I would set this as an H3 or H4 header just so it's clear that the first section of the readme is dedicated to explaining what is Kura

```suggestion
### I have used Eclipse Kura™ to make a small-scale Edge deployment, how do I scale now?
```

File: README.md [hunk 6]
```diff
@@ -15,18 +15,33 @@ Eclipse Kura™
...
+* **Kura™ Networking:** Manage Network connectivity, including 
+* **Kura™ Wires:** Design data flows and data processing streams effortlessly with a drag-and-drop visual editor.
+* **Kura™ Cloud Connectors:** Extendable cloud connector system. 
+* **Kura™ Drivers:** Extendable service that handles reading data off of external devices.
+* **Kura™ Snapshots:** Securely store and re-apply gateway settings for convenience.
+* **Kura™ Security**: Easily and safely store your secrets.
+* **Kura™ Container Orchestrator**: Manage Docker or Podman containers on your gateway for ultimate flexibility.
+* **Kura™ AI Inference**: Run Nvidia Triton Models on the edge.
+* **Kura™ Plugins**: Add and Extend the framework by adding your own Services, and Drivers.
+* **Kura™ REST Service**: Embed the framework as a backend in your own edge applications.
```
Comment (mattdibi) [inline README.md:34]: 🤔 Thinking about it, having all these "Kura™"s is a little redundant maybe? What do you think? Is there a better way? We can also leave this as-is...

Comment (GregoryIvo) [reply README.md:34]: It's a tough call. If eclipse actually holds the kura trademark we should definitely actively protect it. (Or else will loose the rights to it). 

Having all the TM's really makes the list hurt to look at.

Comment (mattdibi) [reply README.md:34]: Actually I was talking about the whole "Kura *" stuff... Like, I know it's "Kura **Services**" we're talking about the Kura project. Would only "Services" or something similar work?

**Review by mattdibi — APPROVED** [review]

**Review by salvatore-coppola — APPROVED** [review]

**Pull request merged**